### PR TITLE
Update ObjectScale 1.2.x End of Standard Support notice for COSI

### PR DESCRIPTION
**Pull request by sharmilarama**
# Description
This PR updates the notice for ObjectScale 1.2.x EOSS as announced in the Dell Support documentation. 

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/1435|

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/cosidriver/_index.md [hunk 1]
```diff
@@ -51,6 +51,8 @@ Dell COSI Driver is a multi-backend driver, meaning that it can connect to multi
 | ObjectScale      |  1.2.x   |
 {{</table>}}
 
+> **NOTE:** Object Scale 1.2.x is planned for End of Standard Support on January 31st 2025. Please refer <a href="https://www.dell.com/support/kbdoc/en-uk/000185734/all-dell-emc-end-of-life-documents?lang=en">Dell Support documentation</a> for more information. We plan to support COSI driver when a new version of ObjectScale is available.
```
Comment (AronAtDell) [inline content/docs/cosidriver/_index.md:54]: Is this "Dell End of Service Life (EOSL)" or "End of Life (EOL)"? If either use this nomenclature as opposed to "End of Standard Support" Thanks!

Comment (sharmilarama) [reply content/docs/cosidriver/_index.md:54]: Is it the EOSS Date as shown in the Notifications spreadsheet in the Dell Support page linked in the note.
image
The spreadsheet elaborates that EOSS is "The date after which Standard Support is not available"

Comment (sharmilarama) [reply content/docs/cosidriver/_index.md:54]: EOL and EOSL dates were not published.

Comment (AronAtDell) [reply content/docs/cosidriver/_index.md:54]: Thanks for the clarification. All set from my side.

**Review by AronAtDell — APPROVED** [review]

**Review by anandrajak1 — APPROVED** [review]

**Pull request merged**